pages/11.miscellaneous/03.plurals/docs.en.md

```diff
@@ -0,0 +1,146 @@
+If **%count%** of computers is more than 1 (5, 13 or 1149) – use **'Template 1'**  
+
+In a situation with another language that has, let’s say, three plural forms, there should be three templates with two vertical bar signs between them. 
+
+For example, we would like to translate this phrase ("Standard license for %count% computer|Standard license for %count% computers") into Slovak, that according to the rule of [Localizations and Plurals](https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals) article has three different forms of words to use with following numbers:
+
+![](https://github.com/AdguardTeam/AdguardKnowledgeBase/blob/plural-forms/pages/11.miscellaneous/plurals/Slovak.png)
+
+Then the translation from English into Slovak should be: 
+
+- Štandartná licencia pre %count% počítač|Štandartná licencia pre %count% počítače|Štandartná licencia pre %count% počítačov 
+
+In this case, we see three templates which contain three plural forms of the word *'počítač'* (computer) in Slovak.
+
+And again, this rule will function during localization proces:
+
+If **%count%** of computers is 1 – use **'Template 0'** –  Štandartná licencia pre  %count% počítač;  
+If **%count%** of computers is more than 1 but less than 5 (2, 3 or 4) – use **'Template 1'** – Štandartná licencia pre  %count% počítače;  
+If **%count%** of computers is 5 or more than 5 – use **'Template 2'** – Štandartná licencia pre  %count% počítačov.
+
+If we ignore one of the forms and use only two templates for Slovak instead of three, the system won’t be able to take an appropriate template for certain numbers and, as a result, we will have grammatical mistakes in sentences, like in English "I have five star".
+
+But in some languages there can be words (let’s call them exclusions) that have fewer plural forms than the common rule of language states words should have. And that can cause some confusion. 
+
+The Serbian word *'računar'* (computer), for example, has only two forms: *'računar'* - for 1, 21, 31 (and other numbers ending in 1, excluding 11) computers and *'računara'* - for other numbers of computers. But in spite of this fact, there should be three templates in AdGuard translations according to the general rule (which states that there are three plural forms of words in Serbian): 
+
+![](https://github.com/AdguardTeam/AdguardKnowledgeBase/blob/plural-forms/pages/11.miscellaneous/plurals/Serbian.png)
+
+Thus, the translation from English ("Standard license for %count% computer|Standard license for %count% computers")  into Serbian in this case should be:
+
+- Standardna licenca za %count% računar|Standardna licenca za %count% računara|Standardna licenca za %count% računara
+
+Although it is obvious that the last two templates have no differences, it is very important to keep the rule: ***The number of templates should reflect the number of plural forms of the respective language!***
+
+***
+
+**Summing up:**
```

Comment:
I think we should make it a separate header and add an anchor here and call it "Short summary" instead of "Summing up"..